rest/data/project.go

```diff
@@ -73,9 +73,12 @@ func RequestS3Creds(ctx context.Context, projectIdentifier, userEmail string) er
 		Project:     settings.ProjectCreation.JiraProject,
 		Summary:     summary,
 		Description: description,
-		Components:  []string{"Access"},
 		Reporter:    userEmail,
+		Fields: map[string]interface{}{
+			evergreen.DevProdServiceFieldName: evergreen.DevProdJiraServiceField,
+		},
```

Comment:
Can we have this be set only for DEVPROD tickets? I can't imagine any other project needs something so specific.